GameServerCore/Maps/IMapScriptHandler.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Numerics;
+using GameServerCore.Content;
+using GameServerCore.Domain;
+using GameServerCore.Domain.GameObjects;
+using GameServerCore.Enums;
+
+namespace GameServerCore.Maps
+{
+    /// <summary>
+    /// Contains all map related game settings such as collision handler, navigation grid, announcer events, and map properties.
```

Comment:
Update summary `Contains all map related game settings such as collision handler, navigation grid, announcer events, and map properties. Doubles as a Handler/Manager for all MapScripts.`